AFQ/utils/path.py

```diff
@@ -0,0 +1,4 @@
+def drop_extension(path):
```

Comment:
This will fail quite horribly on Windows machines (which our previous implementation probably also did...). Maybe use https://docs.python.org/3/library/pathlib.html?

Reply:
And maybe add some tests with Windows style paths? https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats

Reply:
BTW: do you think this helps explain #824 ?

Reply:
I think that is different